### PR TITLE
[CRO] update analytics version

### PR DESCRIPTION
**Pull request by NikitK-deriv**
Changes:

enable google analytics via analytics package for future needs. make pageView as authorized as well

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: gatsby-browser.js [hunk 1]
```diff
@@ -97,8 +96,7 @@ export const onClientEntry = () => {
         device_language: navigator?.language || ' ',
         device_type: isMobile ? 'mobile' : 'desktop',
     })
-    const { tracking } = Analytics.getInstances()
-    tracking.identifyEvent(Analytics?.getId(), { language: getLanguage() })
+    Analytics.identifyEvent()
```
Comment (habib-deriv) [inline gatsby-browser.js:99]: Do we need to check if Analytics is available?

Comment (NikitK-deriv) [reply gatsby-browser.js:99]: valid point, let's make it optional 👍

**Review by habib-deriv — DISMISSED** [review]
LGTM

**Comment by dan-deriv** [comment]
@NikitK-deriv is there a card for this?

**Review by fasihali-deriv — APPROVED** [review]

**Pull request merged**